ldk/goja/src/index.ts

```diff
@@ -6,7 +6,31 @@ import * as network from './network';
 import * as process from './process';
 import * as ui from './ui';
 import * as vault from './vault';
-import * as window from './window';
 import * as whisper from './whisper';
+import * as window from './window';
+
+/*eslint-disable */ // These are provided to validate that the aptitude object meets the interface contract.
+const _clipboard: clipboard.Clipboard = clipboard;
+const _cursor: cursor.Cursor = cursor;
+const _keyboard: keyboard.Keyboard = keyboard;
+const _filesystem: filesystem.Filesystem = filesystem;
+const _network: network.Network = network;
+const _process: process.Process = process;
+const _ui: ui.UI = ui;
+const _vault: vault.Vault = vault;
+const _whisper: whisper.WhisperAptitude = whisper;
+const _window: window.Window = window;
+/*eslint-disable */
 
-export { clipboard, cursor, filesystem, keyboard, network, process, ui, vault, window, whisper };
+export {
+    clipboard,
```

Comment:
won't this till have the clipboard.clipboard problem?

Reply:
@scottkipferOlive - We have tested this out in a few projects, and are seeing the expected behavior. I've attached a few screenshots illustrating how you can import these aptitudes.

![Screen Shot 2021-04-22 at 11 15 02 AM](https://user-images.githubusercontent.com/81329854/115740349-b988c880-a35c-11eb-837e-45dfaefd9ae4.png)
![Screen Shot 2021-04-22 at 11 14 48 AM](https://user-images.githubusercontent.com/81329854/115740361-bd1c4f80-a35c-11eb-937a-20b1baf37f84.png)
![Screen Shot 2021-04-22 at 11 17 43 AM](https://user-images.githubusercontent.com/81329854/115740401-c4dbf400-a35c-11eb-9dc0-801e2b0df6b9.png)

Is there something else we are missing?

Reply:
Looks good